### DISCUSSION TITLE
Random Sort

### DISCUSSION THREAD

**Discussion by mrsmithau** [aashishksahu/SafeSpace#79] [General]
SafeSpace is an excellent app, can you add a random sort of folder? Thanks!

**Comment by aashishksahu** [comment]
Thanks, what's a random sort?

**Reply by mrsmithau** [reply]
Now I can sort files by Name, Date or Size (Ascending / Descending), random may be useful to get random file sequence.

**Reply by mrsmithau** [reply]
shuffle sort...

**Comment by aashishksahu** [comment]
"random sort" and "shuffle sort" is an oxymoron, I'll have to add a database to manage the order which will be a lot of work and refactoring, not sure If implementing this would add much to the functionality. Thanks for the suggestion though, much appreciated :)

**Reply by mrsmithau** [reply]
Thanks for the answer and the excellent software (with or without random sort)
